### ISSUE TITLE
🆕 Software Suggestion | fluffernitter

### ISSUE THREAD

**Issue by aaronfg** [privacytools/privacytools.io#2109]
## Basic Information

**Name:** fluffernitter
**Category:** Nitter redirector app for mobile
**URL:** https://github.com/aaronfg/fluffernitter

## Description
The app registers itself able to handle twitter.com. mobile.twitter.com and t.co links and redirects them to the approproate nitter.net (or any other Nitter instance) page equivalent. Also works by sharing any twitter url to the app

Currently Android only but will be adding iOS in the coming weeks once I've gotten the bugs worked out.

## Why I am making the suggestion

Nitter.net is a popular way privacy-focused people browse twitter. My app helps give the same sort of functionality for auto-redirecting to a nitter instance that you get from using a browser extension. Seems helpful/useful.

## My connection with the software

I am the author of the app.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by wylel** [comment]
> Q: Is this available in the Play Store or F-Droid?

A: Not yet. I'll be getting the Play Store version up hopefully in the next few months. I'm not sure if I'll be publishing to F-Droid at all but we'll see.

Why would you not add it to F-Droid?

**Comment by aaronfg** [comment]
Just was looking over the steps to add it and it seems to be Linux based, checking out and building it? Before you add a PR with your project in there.

As I don't have a Linux setup, F-Droid just seemed to be a "at some point" rather than something I could do immediately.

**Comment by wylel** [comment]
Understandable, but even Linux in a VM could work, or someone could help out (beauty of FOSS).

The reason why I think its important, is a lot of folks on the Android side of Privacy tend to go toward custom, privacy respecting ROMs and is something that is pushed on PTIO (like GrapheneOS which requires the likes of F-Droid, or sideloading) and CalyxOS with the optional microG (without it, apps have to come from other stores like F-Droid) and would be a great addition to an open source app store like it.

**Comment by aaronfg** [comment]
I agree 100%. I always planned on releasing there, I just didn't realize it was such a pain to submit an app. It made it lower on the immediate priority list is all. 

I will update the README to better clarify that an F-Droid release is on the road map. 

I just realized the wording of it as it is now makes it seem like I wasn't sure if I was going to release it there for some other reason besides the practical one. 

I do publish the apks in the repo at least though for now for anyone who doesn't want to build it themselves or can't.

**Comment by lynn-stephenson** [comment]
@aaronfg Why not upload built APKs for each GitHub release you make? Instead of using a folder in your Git repository.

**Comment by aaronfg** [comment]
Upload them where?

**Comment by lynn-stephenson** [comment]
Via GitHub releases. Like a lot of other software projects do. See Aegis Authenticator.

**Comment by aaronfg** [comment]
Ah, I see there's "Assets" for their releases. That is something I didn't even know I could do.

Thanks for letting me know! I'll add that asap.
